pkg/interactive-installer/src/views/mod.rs

```diff
@@ -0,0 +1,27 @@
+/*
+ * rbmenu-tui - RBMenu TUI
```

Comment:
@jsfakian , unfortunately if we want to distribute all EVE sources under Apache 2.0 we cannot use GPL code here.... you will need to find a replacement with a more permissive license (MIT, for instance), or write a alternative version licensed under Apache 2.0....